### DISCUSSION TITLE
Database file

### DISCUSSION THREAD

**Discussion by anthonyraf** [guedesfelipe/pls-cli#71] [Q&A] [answered]
Hi, If I need to backup my tasks or just share them on other machines, is it possible to export and import them again?

I have tried `pls` config and I have the following output:
![image](https://user-images.githubusercontent.com/100955619/235774543-5a58687d-dd43-4758-8c46-b038104710d3.png)

However, I'm not sure if an explorer should appear because I am using Codespace, and I have no graphical interface access.

Thanks.

**Comment by anthonyraf** [comment, marked as answer]
After exploring the source code, I have found that the configuration file is stored at `$HOME/.config/pls/config.json`.
I believe it would be more convenient if we could directly access the config path from the `pls-cli`.

**Reply by guedesfelipe** [reply]
The `pls config` command will only work if it has a graphical interface. I will add a command to show the path in the next version. Thanks a lot for the feedback.

Issue: https://github.com/guedesfelipe/pls-cli/issues/72


If you come up with some way to share the config file with other devices, share it with us here 🤓

**Comment by guedesfelipe** [comment]
Available on [0.4.1](https://github.com/guedesfelipe/pls-cli/releases/tag/0.4.1) 🚀
